package.json

```diff
@@ -20,8 +20,13 @@
     ]
   },
   "resolutions": {
-    "react-native": "0.77.0",
-    "react": "18.3.1",
+    "react-native": "0.78.0",
+    "react-native-safe-area-context": "5.2.0",
+    "react-native-reanimated": "4.0.0-beta.2",
+    "react-native-screens": "4.9.0-beta.0",
+    "react": "19.0.0",
+    "react-dom": "19.0.0",
+    "react-test-renderer": "19.0.0",
```

Comment:
do we need to force all these resolutions? can't we just bump these packages in the whole repo?

Reply:
yes, getting there

Reply:
resolutions are removed